torch/legacy/nn/MarginRankingCriterion.py

```diff
@@ -60,7 +60,7 @@ def updateGradInput(self, input, y):
             self.mask = self.mask.resize_as_(input[0]).copy_(dist)
             mask = self.mask
 
-            torch.ge(dist, 0, out=mask)
+            torch.ge_t(dist, 0, out=mask)
```

Comment:
I don't know if it's used, we only see the core lib, not user code. I agree that what we have now is not the best, but I don't think this is the right solution